atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/ClientLockWatchSnapshot.java

```diff
@@ -61,6 +61,13 @@ private ClientLockWatchSnapshot() {
                 ImmutableSet.copyOf(watches));
     }
 
+    LockWatchStateUpdate.Snapshot getSnapshotWithEvents(LockWatchEvents events, UUID versionId) {
```

Comment:
Check continuity?

Reply:
Factored out a check so that this is always the case (in `ClientLockWatchSnapshot` normally as well as in this case).